.mailmap

```diff
@@ -20,3 +20,11 @@ Felipe Neves <ryukokki.felipe@gmail.com> <ryukokki.felipe@gmail.com>
 Amir Kaplan <amir.kaplan@intel.com> <amir.kaplan@intel.com>
 Anas Nashif <anas.nashif@intel.com> <anas.nashif@intel.com>
 Ruud Derwig <Ruud.Derwig@synopsys.com> <Ruud.Derwig@synopsys.com>
+Flavio Arieta Netto <flavio@exati.com.br>
+Nishikant Nayak <nishikantax.nayak@intel.com>
+Justin Watson <jwatson5@gmail.com>
+Johann Fischer <j.fischer@phytec.de>
+Jun Li <jun.r.li@intel.com>
+Xiaorui Hu <xiaorui.hu@linaro.org>
+Yannis Damigos <giannis.damigos@gmail.com> <ydamigos@iccs.gr>
+Vinayak Kariappa Chettimada <vinayak.kariappa.chettimada@nordicsemi.no> <vinayak.kariappa.chettimada@nordicsemi.no> <vich@nordicsemi.no> <vinayak.kariappa@gmail.com>
```

Comment:
duplicate `<vinayak.kariappa.chettimada@nordicsemi.no>`

Reply:
remove <vinayak.kariappa@gmail.com>

Reply:
looks like you used it  to commit something in the past, and the point of this change to attribute this to you and not consider it some other person.

Reply:
see this 99ae80b6323f96ae33fc5f500614babf4056b823